frontend/test/pytest/test_aws_braket_devices.py

```diff
@@ -0,0 +1,702 @@
+
+        @qjit()
+        @qml.qnode(device)
+        def probs_1qbit(x: float):
+            qml.RX(x, wires=0)
+            return qml.probs()
+
+        observed = probs_1qbit(np.pi / 2)
+        assert np.allclose(np.sum(observed), 1.0)
+
+    @pytest.mark.parametrize(
+        "device",
+        [
+            qml.device(
+                "braket.local.qubit",
+                backend="braket_sv",
+                wires=2,
+                shots=1000,
+            ),
+        ],
+    )
+    def test_probs_on_2qbits_braket(self, device):
+        """Test probs on 2 qubits on braket devices."""
+
+        @qjit()
+        @qml.qnode(device)
+        def probs_2qbits(x: float):
+            qml.RX(x, wires=0)
+            qml.RY(x, wires=1)
+            return qml.probs()
+
+        observed = probs_2qbits(np.pi / 3)
+        assert np.allclose(np.sum(observed), 1.0)
+
+
+@pytest.mark.braket
+class TestBraketCounts:
+    """Unit tests for ``qml.counts``."""
+
+    @pytest.mark.parametrize(
+        "device",
+        [
+            qml.device(
+                "braket.aws.qubit",
+                device_arn="arn:aws:braket:::device/quantum-simulator/amazon/sv1",
+                wires=1,
+                shots=1000,
+            ),
+            qml.device(
+                "braket.local.qubit",
+                backend="braket_sv",
+                wires=1,
+                shots=1000,
+            ),
+        ],
+    )
+    def test_count_on_1qbit_braket(self, device):
+        """Test counts on 1 qubits on braket devices."""
+
+        @qjit()
+        @qml.qnode(device)
+        def counts_1qbit(x: float):
+            qml.RX(x, wires=0)
+            return qml.counts()
+
+        expected = [np.array([0, 1]), np.array([1000, 0])]
+        observed = counts_1qbit(0.0)
+        assert np.array_equal(observed, expected)
+
+        expected = [np.array([0, 1]), np.array([0, 1000])]
+        observed = counts_1qbit(np.pi)
+        assert np.array_equal(observed, expected)
+
+    @pytest.mark.parametrize(
+        "device",
+        [
+            qml.device(
+                "braket.local.qubit",
+                backend="braket_sv",
+                wires=2,
+                shots=1000,
+            ),
+        ],
+    )
+    def test_count_on_2qbits_braket(self, device):
+        """Test counts on 2 qubits on braket devices."""
+
+        @qjit()
+        @qml.qnode(device)
+        def counts_2qbit(x: float):
+            qml.RX(x, wires=0)
+            qml.RY(x, wires=1)
+            return qml.counts()
+
+        expected = [np.array([0, 1, 2, 3]), np.array([1000, 0, 0, 0])]
+        observed = counts_2qbit(0.0)
+        assert np.array_equal(observed, expected)
+
+        expected = [np.array([0, 1, 2, 3]), np.array([0, 0, 0, 1000])]
+        observed = counts_2qbit(np.pi)
+        assert np.array_equal(observed, expected)
+
+
+@pytest.mark.braket
+class TestBraketExpval:
+    """Unit tests for ``qml.expval``."""
```

Comment:
we now have on the roadmap a story for adding support for lowering `expval` for devices that do not support it 👍